tests/SwaggerServiceTest.php

```diff
@@ -683,6 +685,61 @@ public function testAddDataPostRequestWithObjectParams()
         $service->addData($request, $response);
     }
 
+    public function testAddDataWithNotExistsMethodOnController()
+    {
+        $this->mockDriverGetTmpData($this->getJsonFixture('tmp_data_get_user_request'));
+
+        $service = app(SwaggerService::class);
+
+        $request = $this->generateRequest(
+            type: 'get',
+            uri: 'users/{id}/assign-role/{role-id}',
+            data: [
+                'with' => ['role'],
+                'with_likes_count' => true,
+            ],
+            pathParams: [
+                'id' => 1,
+                'role-id' => 5,
+            ],
+            controllerMethod: 'notExists'
+        );
+
+        $response = $this->generateResponse('example_success_user_response.json', 200, [
+            'Content-type' => 'application/json',
+        ]);
+
+        $service->addData($request, $response);
+    }
+
+    public function testAddDataWithBindingInterface()
+    {
+        $this->app->bind(TestContract::class,TestImplementation::class);
+        $this->mockDriverGetTmpData($this->getJsonFixture('tmp_data_get_user_request'));
```

Comment:
for this test case we need to:
- bind some test request to the contract
- get this test request class using trait
- parse rules from it
- assert that rules had been parsed and documentation saved.

The current test implementation just call this method but not assert that it work correcting. Please implement last point by using `mockDriverGetEmptyAndSaveTmpData` mock method

Reply:
Please take a look at the new changes. I'm not sure that I get it right.